gui/screens/onboarding.py

```diff
@@ -376,13 +376,40 @@ def __init__(self):
         self._confirm_password_text_field = TextEdit('mainWindow_passwordViewNewPasswordConfirm')
         self._create_button = Button('mainWindow_Create_password_StatusButton')
         self._password_view_object = QObject('mainWindow_view_PasswordView')
+        self._strength_indicator = QObject('mainWindow_strengthInditactor_StatusPasswordStrengthIndicator')
+        self._indicator_panel_object = QObject('mainWindow_RowLayout')
 
     @property
     @allure.step('Verify: Create password button enabled')
     def is_create_password_button_enabled(self) -> bool:
         return driver.waitForObjectExists(self._create_button.real_name,
                                           configs.timeouts.UI_LOAD_TIMEOUT_MSEC).enabled
 
+    @property
+    @allure.step('Get strength indicator color')
+    def strength_indicator_color(self) -> str:
+        return self._strength_indicator.object.fillColor['name']
+
+    @property
+    @allure.step('Get strength indicator text')
+    def strength_indicator_text(self) -> str:
+        return self._strength_indicator.object.text
+
+    @property
+    @allure.step('Get indicator panel green messages')
+    def green_indicator_messages(self) -> typing.List[str]:
+        messages = []
+        for child in walk_children(self._indicator_panel_object.object):
+            if getattr(child, 'id', '') == 'lowerCaseTxt' and child.color['name'] == '#4ebc60':
```

Comment:
can we move the color code to constants somewhere? I mean `'#4ebc60'` value. In case it changes , would be easy to change as well

Reply:
Yes, you are right, I will

Reply:
Done